MidasCivil_Engine/Compute/FEMeshToPanel.cs

```diff
@@ -35,6 +36,7 @@ public static partial class Compute
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [ToBeRemoved("4.2","Method was replaced in 4.2 by BH.Engine.Structure.Convert.FEMeshToPanel.")]
```

Comment:
Can we raise an issue to remove and version in next milestone to avoid being forgotten until 2030? 😄